docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
-
-   Run the [dotnet restore](../tools/dotnet-restore.md) ([see note](#dotnet-restore-note)) command to restore the dependencies specified in your project.
-
-1. Create a Debug build of your app.
-
-   Use the [dotnet build](../tools/dotnet-build.md) command to build your application or the [dotnet run](../tools/dotnet-run.md) command to build and run it.
-
-1. Deploy your app.
-
-   After you've debugged and tested the program, create the deployment by using the following command:
-
-      ```console
-      dotnet publish -f netcoreapp2.1 -c Release
-      ```
-   This creates a Release (rather than a Debug) version of your app. The resulting files are placed in a directory named *publish*      that's in a subdirectory of your project's *bin* directory.
-
-   Along with your application's files, the publishing process emits a program database (.pdb) file that contains debugging information about your app. The file is useful primarily for debugging exceptions. You can choose not to distribute it with your application's files. You should, however, save it in the event that you want to debug the Release build of your app.
-
-   You can deploy the complete set of application files in any way you like. For example, you can package them in a Zip file, use a simple `copy` command, or deploy them with any installation package of your choice.
-
-1. Run your app
+\* When using this SDK version, this is the default Publish Mode when running the basic `dotnet publish` command.
 
-   Once installed, users can execute your application by using the `dotnet` command and providing the application filename, such as `dotnet fdd.dll`.
+## Publishing basics
 
-   In addition to the application binaries, your installer should also either bundle the shared framework installer or check for it as a prerequisite as part of the application installation.  Installation of the shared framework requires Administrator/root access.
+When you publish your app, the default target the app is compiled for (automatically chosen when you create the app) is specified in the `<TargetFramework>` setting of the project file. Choose any valid [Target Framework Moniker (TFM)](~/dotnet/standard/frameworks.md). For example, if you have `<TargetFramework>netcoreapp2.2</TargetFramework>` chosen, an executable that targets .NET Core 2.2 is created. The TFM specified in this setting is the default target used by the [`dotnet publish`][dotnet-publish] command.
 
-## Framework-dependent deployment with third-party dependencies
+You can use the `<TargetFrameworks>` setting (note the plural) to specify more than one framework supported by your app. Separate each TFM with a semicolon. You can publish, targeting one of the frameworks listed, with the `dotnet publish -f <TFM>` command. For example, if you have `<TargetFrameworks>netcoreapp2.1;netcoreapp2.2</TargetFrameworks>`, and run `dotnet publish -f netcoreapp2.1`, an executable that targets .NET Core 2.1 is created. 
```

Comment:
- The "targeting..." parenthetical is awkwared. "When you publish, you target one of the frameworks listed with the...
- no comma before "and run"